druid/examples/hello_wasm_web/README.md

```diff
@@ -0,0 +1,29 @@
+# druid WASM hello world
+
+This is a minimal example of building a single druid application for the web. To build all the druid examples as wasm, check out the main wasm example.
+
+## Building
+
+You will need `cargo` and `wasm-pack` for building the code and a simple
+server like [`http`](https://crates.io/crates/https) for serving the web page.
+
+First build with.
+
+```
+> wasm-pack build --target web
```

Comment:
It might be nice to add `--dev` here, as `wasm-pack` builds in release mode by default.